samplejava/src/main/java/mx/leo/sample/easyrecycler/view/SampleHeaderView.java

```diff
@@ -0,0 +1,14 @@
+package mx.leo.sample.easyrecycler.view;
+
+import java.util.List;
+
+import mx.leo.sample.easyrecycler.model.Language;
+
+/**
+ * Created by Braulio on 22/12/2016.
+ */
+
+public interface SampleHeaderView {
+
+    void setItems(List<Language> languages);
```

Comment:
It's better to use actions to create methods into views 
Example
ShowItems instead of setItems